model-optimizer/unit_tests/mo/front/tf/extractors/utils_test.py

```diff
@@ -211,3 +211,17 @@ def test_str_decode_list(self):
         with self.assertLogs(log.getLogger(), level="ERROR") as cm:
             result = tf_tensor_content(pb_tensor.dtype, shape, pb_tensor)
             self.assertEqual([warning_message, warning_message], cm.output)
+
```

Comment:
it seems that we have no case with a scalar. For example, 
        pb_tensor = PB({
            'dtype': 3,
            'int_val': 4,
        })
Please add it.